http/1.7/modules/ROOT/pages/http-authentication.adoc

```diff
@@ -699,6 +700,223 @@ In the *Configuration XML* editor, the `tls:context`, `tls:trust-store` and `tls
     </tls:context>
 ----
 
+[[oauth2-grant-authentication-connected-apps]]
+== Configure OAuth2 Authorization Code Grant Type Authentication Using Connected Apps
+:page-aliases: connectors::http/http-configure-authorization-code-for-requester.adoc
+
+You can also configure the OAuth2 authorization code grant type authentication using the xref:access-management::connected-apps-developers.adoc[Connected Apps] feature.  
+
+Thw following example shows how to configure the OAuth2 authorization code grant type authentication for the HTTP *Request* operation by creating a Mule app to access a protected resource and the resource in the resource server. See xref:mule-runtime::authorization-grant-types.adoc[OAuth Authorization Grant Types]. The example covers how to:
+
+* Create the Mule app
+* Add redirection to local authorization URL
+* Add OAuth authentication
+* Add the connected app
+* Add token URL
+* Run the Mule app
+
+This example requires that you have an Anypoint Platform account that has the organization administrator permission.
+
+Before you create the Mule app, review the following steps for getting OAuth access token and returning token for data:
+
+. The user submits an HTTP request in the Mule app.
+. The Mule app redirects the request to the protected resource in the Anypoint organization server.
+. The Anypoint organization server rejects the request because the Mule app doesn't have a token.
+. The Mule app redirects the user to an URL corresponding to the authorization server, indicating where to call the Mule app back when the authorization code is available.
+. The user authenticates itself to the authorization server.
+. The authorization server returns an authorization code back to the redirect URI.
+. The Mule app calls the authorization server again and provides the authorization code to obtain an access token (stored in a object store).
+. The Mule app calls the authorization server, providing a valid access token.
+. The Anypoint organization server validated the token and delivers the protected resource after successful authentication. 
++
+Note that the user is not expose to the token nor the Mule app sees customer credentials, but just an authorization code. 
+
+=== Create the Mule App
+
+When you create a Mule app, the flow tries to access a protected resource in the Anypoint organization server with no authentication configured:
+
+[source,xml,linenums]
+----
+<http:listener-config name="listenerConfig">
+  <http:listener-connection host="0.0.0.0" port="8070" />
+</http:listener-config>
+
```

Comment:
```suggestion
<configuration-properties file="local-config.yaml" />
```

Reply:
We could add a reference to this document: https://docs.mulesoft.com/mule-runtime/4.4/configuring-properties
I think that a customer who is configuring OAuth would already know how to configure properties, but it's a precondition and we could just add the link...

Reply:
I've added this reference to the section "This example requires that you..."